.eslintrc.js

```diff
@@ -21,6 +21,7 @@ module.exports = {
         'react/jsx-props-no-spreading': 0,
         'react/require-default-props': 0,
         'class-methods-use-this': 0,
+        'jsx-a11y/no-autofocus': 0,
```

Comment:
```suggestion
        'jsx-a11y/no-autofocus': 0,
        "import/no-extraneous-dependencies": ["error", {"devDependencies": ["**/*.stories.tsx"]}],
```
If you want to avoid ignoring that rule in all stories.